apps/dashboard/app/helpers/scripts_helper.rb

```diff
@@ -4,6 +4,8 @@
 module ScriptsHelper
   def create_editable_widget(form, attrib, format: nil)
     widget = attrib.widget
+    name = "#{form.object_name}[#{attrib.id}][value]"
+    #attrib.html_options = { class: 'real-field', autocomplete: 'off', name: name }
```

Comment:
Please ignore this part, I was doing some testing on submitting the form parameters as a hash to avoid the `field_property` notation used for exclude/min/max/fixed